doc/source/running.rst

```diff
@@ -26,13 +26,21 @@ sure one instance of it is started for each mongod node.
 
 E.g. Imagine you put configsvr nodes (listen port 27019) colocated on the same
 servers as the first shard's mongod nodes (listen port 27018, replica set name
-"sh1rs") to save some hardware costs. In this server you would start two
+"sh1rs"). In this server there should be two 
 |pbm-agent| processes, one connected to the shard
 (e.g. "mongodb://username:password@localhost:27018/") and one to the configsvr
 node (e.g. "mongodb://username:password@localhost:27019/").
 
-It is best to use the packaged service scripts to run |pbm-agent|. But for
-reference an example of how to do it manually is shown below. The output is
+It is best to use the packaged service scripts to run |pbm-agent|. After
+adding the database connection configuration for them (see pbm.installation.service_init_scripts_) 
+you can start the |pbm-agent| service as below:
+
+.. code-block:: bash
+
+   sudo systemctl start pbm-agent
+   sudo systemctl status pbm-agent
+
+For reference an example of how to do it manually is shown below. The output is
```

Comment:
"do" and "it" are not specific terms. the passive voice here ("is shown")  makes it harder to follow the explanation:
The following example demonstrates how you can start |pbm-agent| manually. The process runs in the background and the output is redirected to a file.

Reply:
OK, fix coming in small patch in a second.

Reply:
Done